taxcalc/utils.py

```diff
@@ -263,33 +269,36 @@ def results(c):
 
 
 def weighted_avg_allcols(df, cols):
-    diff = DataFrame(df.groupby('bins').apply(weighted_mean, "c00100"),
-                     columns=['c00100'])
+    diff = DataFrame(df.groupby('bins')['s006'].sum(), columns=['s006'])
```

Comment:
I am a bit stuck here. The general idea of this function was to start with with weighted average of the amount of tax paid (`c00100`), and then add columns to that DataFrame by doing the weighted average of the rest of the columns in TABLE_LABELS. But now, `diff` starts out as the weighted average of `s006` which are the weights themselves. Can you help me understand why we want to see the weighted average of `s006` here?